### PR TITLE
Network policies

### PR DESCRIPTION
**Pull request by fdistorted**
fixes cloudquery/cloudquery-issues#128

### CODE REVIEW

File: resources/networking_network_policies.go [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+			{
+				Name:        "policy_types",
+				Description: "List of rule types that the NetworkPolicy relates to. Valid options are [\"Ingress\"], [\"Egress\"], or [\"Ingress\", \"Egress\"]. If this field is not specified, it will default based on the existence of Ingress or Egress rules; policies that contain an Egress section are assumed to affect Egress, and all policies (whether or not they contain an Ingress section) are assumed to affect Ingress. If you want to write an egress-only policy, you must explicitly specify policyTypes [ \"Egress\" ]. Likewise, if you want to write a policy that specifies that no egress is allowed, you must specify a policyTypes value that include \"Egress\" (since such a policy would not include an Egress section and would otherwise default to just [ \"Ingress\" ]). This field is beta-level in 1.8",
+				Type:        schema.TypeStringArray,
+				Resolver:    schema.PathResolver("Spec.PolicyTypes"),
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "k8s_networking_network_policy_owner_references",
```
Comment (roneli) [inline resources/networking_network_policies.go:118]: this should become a json

File: resources/networking_network_policies.go [hunk 2]
```diff
@@ -0,0 +1,663 @@
...
+					},
+					{
+						Name:        "block_owner_deletion",
+						Description: "If true, AND if the owner has the \"foregroundDeletion\" finalizer, then the owner cannot be deleted from the key-value store until this reference is removed. Defaults to false. To set this field, a user needs \"delete\" permission of the owner, otherwise 422 (Unprocessable Entity) will be returned.",
+						Type:        schema.TypeBool,
+					},
+				},
+			},
+			{
+				Name:        "k8s_networking_network_policy_managed_fields",
```
Comment (roneli) [inline resources/networking_network_policies.go:163]: this should become a json

File: resources/networking_network_policies.go [hunk 3]
```diff
@@ -0,0 +1,663 @@
...
+							{
+								Name:        "ip_block_except",
+								Description: "Except is a slice of CIDRs that should not be included within an IP Block Valid examples are \"192.168.1.1/24\" or \"2001:db9::/64\" Except values will be rejected if they are outside the CIDR range",
+								Type:        schema.TypeStringArray,
+								Resolver:    schema.PathResolver("IPBlock.Except"),
+							},
+						},
+						Relations: []*schema.Table{
+							{
+								Name:        "k8s_networking_network_policy_ingress_from_pod_selector_match_expressions",
```
Comment (roneli) [inline resources/networking_network_policies.go:318]: json

File: resources/networking_network_policies.go [hunk 4]
```diff
@@ -0,0 +1,663 @@
...
+									{
+										Name:        "values",
+										Description: "values is an array of string values",
+										Type:        schema.TypeStringArray,
+									},
+								},
+							},
+							{
+								Name:        "k8s_networking_network_policy_ingress_from_namespace_selector_match_expressions",
+								Description: "A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.",
```
Comment (roneli) [inline resources/networking_network_policies.go:347]: json

File: resources/networking_network_policies.go [hunk 5]
```diff
@@ -0,0 +1,663 @@
...
+										Type:        schema.TypeStringArray,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			{
+				Name:        "k8s_networking_network_policy_egress",
```
Comment (roneli) [inline resources/networking_network_policies.go:378]: same ingress above I think an extra table was created

**Review by roneli — CHANGES_REQUESTED** [review]
Some required changes in the tables is needed

File: resources/networking_network_policies.go (hunk 5 above)
Comment (fdistorted) [reply resources/networking_network_policies.go:261]: there is ingress and egress - they are different

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**